### ISSUE TITLE
[Authentication code] Step may not be descriptive

### ISSUE THREAD

**Issue by svinkle** [CovidShield/portal#66]
## Issue summary

Step 2 of the instructions may not be descriptive enough to convey clear instructions.

> white section at the bottom of the screen

This does not explain the step in a way which is informative to a screen reader user. This could lead to a confusing user experience.

<img width="497" alt="Screen Shot 2020-05-25 at 1 35 22 PM" src="https://user-images.githubusercontent.com/1392632/82833657-a5a6bf00-9e8c-11ea-8fa1-5a48dc0f1574.png">

#### HTML

```html
<li class="ordered-list__item">
  <div class="ordered-list__item-content">
    <span>Tap the white section at the bottom of the screen.</span>
    <!-- … -->
  </div>
</li>
```

## Steps to reproduce

1. Go to https://portal-rails.myshopify.io/ with with any supported browser
2. Login with `admin`/`password`
3. With an active screen reader, navigate through the page content via virtual cursor

### Expected

- Each step to be clear and precise.

### Actual

- Step 2 content may not be clear.

## Recommendation

Adjust the step content to include the accessible name of the Sheet control.

#### HTML

```html
<li class="ordered-list__item">
  <div class="ordered-list__item-content">
    <span>Tap the COVID Shield Status <strong>button</strong> at the bottom of the screen.</span>
    <!-- … -->
  </div>
</li>
```

## Specifications

- **Component:** Authentication code
- **Related:** https://github.com/CovidShield/mobile/issues/37
- **WCAG Principle:** Operable
- **WCAG SC:** 2.4.6 Headings and Labels
- **Severity:** Medium
- **Effort:** Low

**Comment by dfmcphee** [comment]
cc @emilybulger

**Comment by emilybulger** [comment]
@svinkle do you have a strong preference for "button" over "section"?

I think most people expect a button to be a rectangle with curved edges, and the bottom sheet isn't quite that.

Proposal:

`Tap the COVID Shield status <strong>section</strong> at the bottom of the screen.`

**Comment by svinkle** [comment]
@emilybulger good point; this instruction may not be clear for sighted users. I recommended the descriptor, _"button"_, as this is the `role` for the control, how it will be describe for screen reader users. But, this isn't actually inclusive.

In the PR https://github.com/CovidShield/mobile/pull/54#issuecomment-635338510 I make the recommendation to change the control label due to [2.5.3 Label in Name](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html), so "Open" is no longer on the table.

From what I understand, the control in question activates a "Sheet", but this isn't common language to use. I'm not sure if "section" is an appropriate descriptor, either. However, "at the bottom of the screen" does help guide the user.

Let's go with `Tap the <strong>COVID Shield status section</strong> at the bottom of the screen.` for the visual text.

@dfmcphee Is it possible to include [`.sr-only`](https://github.com/h5bp/html5-boilerplate/blob/master/dist/css/main.css#L112) like style definition for a `Text` component? I'm thinking it might be helpful to have a hidden descriptor for screen reader users to consume. Thoughts?

**Comment by svinkle** [comment]
@dfmcphee Clearly I've been in React Native land too long and forgot this was the Portal. 😅

I'm thinking something like:

```html
Tap the <strong>COVID Shield status <span aria-hidden="true">section</span><span class="sr-only">button</span></strong> at the bottom of the screen.
```
